samples/client/petstore/csharp-netcore/OpenAPIClient-ConditionalSerialization/src/Org.OpenAPITools/Model/FormatTest.cs

```diff
@@ -716,18 +716,9 @@ public override int GetHashCode()
                 {
                     hashCode = (hashCode * 59) + this.Binary.GetHashCode();
                 }
-                if (this.Date != null)
-                {
-                    hashCode = (hashCode * 59) + this.Date.GetHashCode();
-                }
-                if (this.DateTime != null)
-                {
-                    hashCode = (hashCode * 59) + this.DateTime.GetHashCode();
-                }
-                if (this.Uuid != null)
-                {
-                    hashCode = (hashCode * 59) + this.Uuid.GetHashCode();
-                }
+                hashCode = (hashCode * 59) + this.Date.GetHashCode();
+                hashCode = (hashCode * 59) + this.DateTime.GetHashCode();
+                hashCode = (hashCode * 59) + this.Uuid.GetHashCode();
```

Comment:
Regarding @wing328 comments, @devhl-labs can you confirm that Uuid are indeed value type and that there should not be null check?

Reply:
Will wait for your review. In the end, if the old `x-is-value-type` is still a thing that need to exist, I can put it back but I'm pretty sure I will keep using only `x-csharp-value-type` in the code.
I will read this later today.
Thanks